test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -264,6 +290,38 @@ test
     await t.expect(loopbackSuccessForChallengeTimeoutLogger.contains(record => record.request.url.match(/\/idp\/idx\/authenticators\/poll\/cancel/))).eql(false);
   });
 
+test
+  .requestHooks(loopbackSuccessLogger, loopbackSuccessButNotAssignedAppMock)('loopback succeeds but user is not assigned to app, then clicks cancel link', async t => {
+    const deviceChallengePollPageObject = await setup(t);
+    await t.expect(deviceChallengePollPageObject.getBeaconClass()).contains(BEACON_CLASS);
```

Comment:
you don't need to repeat checks that are tested already in previous tests.. keep the test short specific to what it intends.